dataactvalidator/validation_handlers/validator.py

```diff
@@ -846,26 +847,41 @@ def validateFileBySql(submissionId,fileType,interfaces):
              values in fields involved
              row number
         """
-        # Skip Sql validation
-        return []
         # Pull all SQL rules for this file type
         fileId = interfaces.validationDb.getFileId(fileType)
         rules = interfaces.validationDb.session.query(RuleSql).filter(RuleSql.file_id == fileId).filter(RuleSql.rule_cross_file_flag == False).all()
         errors = []
         # For each rule, execute sql for rule
         for rule in rules:
-            failures = interfaces.validationDb.connection.execute(rule.rule_sql.format(submissionId))
+            failures = interfaces.stagingDb.connection.execute(rule.rule_sql.format(submissionId))
             # Build error list
             for failure in failures:
                 row = failure["row_number"]
-                errorMsg = rule.rule_error_msg
-                values = ""
-                fieldNames = ""
-                for field in failure:
+                errorMsg = rule.rule_error_message
+                values = None
+                fieldNames = None
+                # failure is a RowProxy, which does not quite act the same as a dict
+                for field in failure.keys():
                     if field == "row_number":
                         # Row handled separately, skip
                         continue
-                    values = ", ".join([values, "{}: {}".format(field,failure[field])])
-                    fieldNames = ", ".join([fieldNames,field])
+                    if values is None:
+                        values = "{}: {}".format(field,failure[field])
+                    else:
+                        values = ", ".join([values, "{}: {}".format(field,failure[field])])
+                    if fieldNames is None:
+                        fieldNames = field
+                    else:
+                        fieldNames = ", ".join([fieldNames,field])
                 errors.append([fieldNames,errorMsg,values,row])
+            # Pull where clause out of rule
```

Comment:
I put a note to our future selves about this: https://federal-spending-transparency.atlassian.net/browse/DB-708
(no action necessary, was just reminded again that we shouldn't be doing this long-term)